server/Dockerfile

```diff
@@ -1,20 +1,27 @@
-FROM openmicroscopy/devslave-c7:0.6.1
+FROM openmicroscopy/devslave-c7:0.7.1
 
 MAINTAINER OME
 
 ARG JAVAVER=openjdk1.8
 ARG ICEVER=ice36
+ARG BRANCH=test
+
+# Remove the existing omero user
+RUN userdel -r omero && \
```

Comment:
As a RFE, is there any other place that depends on https://github.com/ome/omero-ssh-daemon-c7-docker/blob/1743c4ca0731b221aafc486d64f175a663b41233/Dockerfile#L16-L19 ? Or should we simply remove this user creation from the base image?

Reply:
several places assume that the omero user exists so I found it safer to remove it.

Reply:
:+1: I can turn this into an issue